Daniel-Bennett777/Buildo-Client
# added styling

**TO TEST**

-git fetch in main
-DUM branch checkout and then make sure your backend is served and shows the new background images throughout the ui
